products/replay/backend/prompts.py

```diff
@@ -2,6 +2,222 @@
 import json
 from datetime import datetime
 
+SESSION_REPLAY_RESPONSE_FORMATS_PROMPT = """
+<response_formats>
+Formats of responses
+1. Question Response Format
+When you need clarification or determines that additional information is required, you should return a response in the following format:
+{
+    "result": "question",
+    "data": {
+        "question": "Your clarifying question here."
+    }
+}
```

Comment:
Should we be instructing to use tool calls here? This sounds like just plain JSON output